src/scroll-timeline-base.js

```diff
@@ -447,39 +448,39 @@ function range(timeline, phase) {
     containerSize = container.clientHeight;
   }
 
-  const scrollPos = directionAwareScrollOffset(container, orientation);
+  const inset = parseInset(details.inset, containerSize);
   let startOffset = undefined;
   let endOffset = undefined;
 
   switch(phase) {
     case 'cover':
       // Range of scroll offsets where the subject element intersects the
       // source's viewport.
```

Comment:
These comments no longer align with the equation.  My preference is to change this to read "adjusted" viewport and update the viewPos and containerSize ahead of the switch to reflect the adjusted viewport.  Then all of the equations within the switch remain unchanged.